wgpu-types/src/instance.rs

```diff
@@ -98,6 +98,11 @@ bitflags::bitflags! {
         ///
         /// When `Self::from_env()` is used takes value from `WGPU_GPU_BASED_VALIDATION` environment variable.
         const GPU_BASED_VALIDATION = 1 << 4;
+
+        /// Validate indirect buffer content prior to issuing indirect draws/dispatches.
+        ///
+        /// When `Self::from_env()` is used takes value from `WGPU_VALIDATION_INDIRECT_CALL` environment variable.
+        const VALIDATION_INDIRECT_CALL = 1 << 5;
```

Comment:
I have thoughts on how this isn't the right place to put this - but that is _not_ this PR's to solve :)

Reply:
Filed https://github.com/gfx-rs/wgpu/issues/7261